itests/org.jboss.tools.cdk.ui.bot.test/README.md

```diff
@@ -0,0 +1,30 @@
+# CDK tooling integration tests
+This plugin contains CDK tooling integration tests. For execution of a specific tests see guidelines below.
+
+## Requirements
+To be able to run CDK integration tests, couple of pre-requisities must be fulfilled.
+1. You need to have an account on https://developers.redhat.com
+2. Have downloaded and configured Container Development Kit (2.x and 3.x). Instructions to download/setup CDK 2.x/3.x can be found here: https://developers.redhat.com/products/cdk/overview/. Even though, there is always supported latest CDK version (3.2 version), integration tests for CDK tooling are still testing proper UI for CDK 2.x and CDK 3.1 adapters. Please, follow the requirements below, you need to have:
+- downloaded CDK 2.x vagrant file
+- downloaded and configured CDK 3.1 binary, `minishift setup-cdk` must be called before test is run in IDE
+- TODO: downloaded and configured CDK 3.2 binary, and have called setup-cdk on it (which creates ~/.minishift folder, and prepares cdk to be able to start from IDE)
+3. For successful registration of vm-guest rhel image your account must have signed Terms and Conditions (https://developers.redhat.com/terms-and-conditions/).
+4. For ease of configuration and automation there is a possibility to use cdk configuration scripts (https://github.com/odockal/cdk-scripts)
```

Comment:
Yes, I can definitely add some examples of how to use those scripts. Just for information, there is read-me file in that repo, I am just working on updating scripts to work with minishift profiles, and then I will update read-me as well. but I will add some scenarios into integration tests anyway, cause this will help to promote my scripts :)